src/components/NavBase.vue

```diff
@@ -615,6 +602,7 @@ $content-max-width: map-deep-get($breakpoint-attributes, (nav, large, content-wi
   @include font-styles(nav-menu);
   flex: 1 1 auto;
   display: flex;
+  justify-content: flex-end;
```

Comment:
This new rule might make sense as a base/default, but I think it is causing a minor regression with the layout of the tutorials navbar where the nav dropdowns are now shifted to the right whereas they were previously lined up with the title area on the left hand side.

We may need to override this in the tutorial navbar now.

Example:
<img width="998" alt="Screenshot 2024-01-29 at 1 22 52 PM" src="https://github.com/apple/swift-docc-render/assets/212918/f57077f2-230e-40e1-b589-c21654810b4e">

Reply:
Good catch!
Fixed here: https://github.com/apple/swift-docc-render/pull/772/commits/a62fcda6081c7b6858884ce3a8213f5064387eaa